python/loan_application/repo/merchant/api.py

```diff
@@ -24,3 +25,27 @@ class _MerchantConfigurationsRepo:
 def get_merchant_configuration(
         merchant_id: str) -> Optional[MerchantConfiguration]:
     return _REPO.repo.get(merchant_id)
+
+'''
+Update method to configure existing merchants with prequal enabled and new min
+and max loan values.
+'''
+def update_merchant_prequal_configuration(
+        merchant_id: str,
+        minimum_loan_amount: Decimal,
+        maximum_loan_amount: Decimal,
+        prequal_enabled: bool,
+) -> Optional[str]:
+    #retrieve merchant from repo
+    merchant_to_update = get_merchant_configuration(merchant_id)
+    if merchant_to_update:
+        #after checking that the merchant is in the repo, update parameters
```

Comment:
I think you should have checks here to ensure that the `max/min` loan are valid. Just like in the `implementation.py` that calls this. 

Actually if you have to choose one place to include it, I would recommend including that check here. 

This is because if another function calls `update_merchant` it should expect it to fail for invalid inputs. I would recommend putting the validation here.